### PR TITLE
[ZT] WARP client logs - add new FAQ

### PR DESCRIPTION
**Pull request by ranbel**
https://github.com/cloudflare/cloudflare-docs/pull/4023

### CODE REVIEW

File: content/cloudflare-one/faq/cloudflare-tunnels-faq.md [hunk 1]
```diff
@@ -121,3 +121,31 @@ If you are having an issue with one or more tunnels, before contacting the Cloud
 - Make sure that the `cloudflared daemon` is updated to the [latest version](https://github.com/cloudflare/cloudflared).
 
 - Gather any relevant error/access logs from your server.
+
+### I am having issues accessing resources through WARP
```
Comment (abracchi-tw) [inline content/cloudflare-one/faq/cloudflare-tunnels-faq.md]:
```suggestion
### I am having issues accessing resources on private networks through the WARP client.
```
as a separate note, do you think it makes sense to phrase this in the first person? I'm thinking those who will have issues in this case will likely be end users, not admins themselves

File: content/cloudflare-one/faq/cloudflare-tunnels-faq.md [hunk 2]
```diff
@@ -100,11 +100,11 @@ $ cloudflared tunnel run
 Tunnel credentials file '/root/.cloudflared/928655cc-7f95-43f2-8539-2aba6cf3592d.json' doesn't exist or is not a file
 ```
 
+## How do I contact support?
 
+### I am having an issue with a locally-managed tunnel
```
Comment (abracchi-tw) [inline content/cloudflare-one/faq/cloudflare-tunnels-faq.md]:
```suggestion
### I am having an issue with a locally-managed tunnel.
```

File: content/cloudflare-one/faq/cloudflare-tunnels-faq.md [hunk 3]
```diff
@@ -121,3 +121,31 @@ If you are having an issue with one or more tunnels, before contacting the Cloud
 - Make sure that the `cloudflared daemon` is updated to the [latest version](https://github.com/cloudflare/cloudflared).
 
 - Gather any relevant error/access logs from your server.
+
+### I am having issues accessing resources through WARP
+
+While [connecting private networks](/cloudflare-one/connections/connect-networks/private-net/) with Cloudflare Tunnels, end users will connect to your resources using the [WARP client](/cloudflare-one/connections/connect-devices/warp/).
```
Comment (abracchi-tw) [inline content/cloudflare-one/faq/cloudflare-tunnels-faq.md]:
```suggestion
If your end users are experiencing issues while accessing resources on private networks [exposed via Cloudflare Tunnel](/cloudflare-one/connections/connect-networks/private-net/), ensure you include WARP client logs when contacting Cloudflare support. These logs will help Cloudflare support understand the overall architecture of your machine and networks.
```

File: content/cloudflare-one/faq/cloudflare-tunnels-faq.md [hunk 4]
```diff
@@ -121,3 +121,31 @@ If you are having an issue with one or more tunnels, before contacting the Cloud
...
+In order to help Cloudflare support understand the overall architecture of your machine and networks, send the WARP Client logs with your request.
+
+#### macOS
```
Comment (abracchi-tw) [inline content/cloudflare-one/faq/cloudflare-tunnels-faq.md]:
```suggestion
#### Retrieving WARP client logs on macOS
```

Comment (abracchi-tw) [reply content/cloudflare-one/faq/cloudflare-tunnels-faq.md]: actually — should we have a separate section under the WARP docs outlining how to retrieve logs and then link out to that section from here? looks like information that users may find helpful regardless of tunnel issues

Comment (ranbel) [reply content/cloudflare-one/faq/cloudflare-tunnels-faq.md]: I was debating whether to put this in the WARP FAQ or Tunnel FAQ. I don't know if the logs are helpful for anything else besides Support issues.

**Review by abracchi-tw — APPROVED** [review]

File: content/cloudflare-one/faq/cloudflare-tunnels-faq.md (hunk 4 above)
Comment (abracchi-tw) [reply content/cloudflare-one/faq/cloudflare-tunnels-faq.md]: yeah, might be worth moving the logs info to the WARP FAQ and linking out to them. Surely users will look for logs-related info for WARP issues outside of tunnel scenarios

**Pull request merged**